File: Python/1117.py
i = 0
med = 0
while i < 2:
    nota = float(input())
    if nota >= 0 and nota <= 10:
        i = i + 1
        med = med + nota
    if nota < 0 or nota > 10:
        print('nota invalida')
med = med / 2
print('media = {:.2f}'.format(med))

File: CSharp/1117.cs
using System; 
using System.Collections.Generic;
using System.Globalization;
using System.Linq;


class URI {

    static void Main(string[] args) 
    {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        List<decimal> listao = new List<decimal>();
        decimal entrada;

        while(listao.Count<2)
        {
            entrada = decimal.Parse(Console.ReadLine());
            if(entrada>=0 && entrada<=10)
            {
                listao.Add(entrada);
            }
            else{Console.WriteLine("nota invalida");}

        }
        Console.WriteLine($"media = {listao.Average():F2}");

    }

}

